java/src/test/android/local.properties

```diff
@@ -0,0 +1,10 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file should *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
```

Comment:
Looks like some stuff here isn't intended to be checked in. Is the '.gitignore' file in this directory meant to apply?

Also wondering if the files ending in '.lock' are meant to be checked in.